test/file-matching.js

```diff
@@ -5,13 +5,13 @@ const path = require('path');
 const tap = require('tap');
 
 const sameMembers = (assert, actual, expected) => {
-  const expectedSet = new Set(expected.map((expectedMember) => {
+  const expectedSet = expected.map((expectedMember) => {
     return expectedMember.split('/').join(path.sep);
-  }));
+  });
   assert.equal(actual.length, expected.length);
 
   for (let actualMember of actual) {
-    assert.assert(expectedSet.has(actualMember), actualMember);
+    assert.assert(expectedSet.includes(actualMember), actualMember);
```

Comment:
From Line 8-14, what is this change?

Reply:
oops, I didn't mean to ship this together, I was just sanity checking we could use an array here - from expected.map on line 8 - with no need to create a Set out of the expected values.

Do you prefer if I revert this?

Reply:
I don't think it's a problem to keep it.